tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                    "thickness": 0.3,
+                    "conductivity": 2.31,
+                    "density": 2322.0,
+                    "specific_heat": 832.0,
+                    "thermal_absorptance": 0.95,
+                    "solar_absorptance": 0.75,
+                    "visible_absorptance": 0.8
+                }
+            ]
+        }
+    },
+    "rooms": [
+        {
+            "type": "RoomAbridged",
+            "name": "TinyHouseZone",
+            "name_original": "Tiny House Zone",
+            "properties": {
+                "type": "RoomProperties",
+                "energy": {
+                    "type": "RoomEnergyPropertiesAbridged",
+                    "program_type": null,
+                    "construction_set": null,
+                    "people": null,
+                    "lighting": null,
+                    "electric_equipment": null,
+                    "gas_equipment": null,
+                    "infiltration": null,
+                    "ventilation": null
+                }
+            },
+            "faces": [
+                "TinyHouseZone_Bottom",
+                "TinyHouseZone_Front",
+                "TinyHouseZone_Right",
+                "TinyHouseZone_Back",
+                "TinyHouseZone_Left",
+                "TinyHouseZone_Top"
+            ],
+            "indoor_shades": [
+                "TinyHouseZone_Back_InShd0"
+            ],
+            "outdoor_shades": [
+                "TinyHouseZone_Back_OutShd0"
+            ]
+        }
+    ],
+    "faces": [
+        {
+            "type": "FaceAbridged",
+            "name": "TinyHouseZone_Bottom",
+            "name_original": "Tiny House Zone_Bottom",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        0.0,
+                        0.0,
+                        0.0
+                    ],
+                    [
+                        0.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        0.0,
+                        0.0
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "FaceProperties",
+                "energy": {
+                    "type": "FaceEnergyPropertiesAbridged",
+                    "construction": "Thermal Mass Floor"
+                }
+            },
+            "face_type": {
+                "type": "Floor"
+            },
+            "boundary_condition": {
+                "type": "Ground",
+                "sun_exposure": "NoSun",
+                "wind_exposure": "NoWind",
+                "view_factor": "autocalculate"
+            },
+            "parent": "TinyHouseZone"
```

Comment:
Suggest having the child reference the parent or the parent reference the children.  Having two way references can lead to data being out of sync.

Reply:
Note, you can have the two way link in memory when you are working with live Python objects.  I just wouldn't write both out to disk.

Reply:
@mostaphaRoudsari  and I have had long conversations about this, which have also included some input from Theo , who provided feedback in terms of what he thought was easiest for editing the schema.  Let's plan to have a longer discussion about this on Wednesday.

Reply:
I am just making a note of @macumber 's suggestion here if we go for the 2-way reference:

> my input on parents referencing children and children referencing parents, this is a fine convenience to serialize to JSON, I would just write a comment on one of the fields in the schema to say "this one is used in deserialization, the other one is not",  that way it is clear which one of the links actually describes the information